tg.Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG TF_BASE_TAG
+FROM dockerhub.tax.service.gov.uk/alpine/terragrunt:${TF_BASE_TAG}
```

Comment:
We use debian-slim as a base for the repo image but alpine here. Is there a specific reason or just a historical artefact type thing?

Reply:
It is more the latter. This image is like a utility/throwaway tool only used for running terraform/terragrunt and aws-cli commands. It is not persisted in ECR.